data/scenarios/Tutorials/move.yaml

```diff
@@ -13,7 +13,7 @@ objectives:
     condition: |
       r <- robotNamed "check";
       w <- as r {count "Win"};
-      return (w >= 2)
+      return (w >= 1)
```

Comment:
@byorgey, thanks for pointing it out; I made a typo while changing the conditions to use one robot.